File: windows/MoDi.Desktop/TransportIdentity.cs
using System;

namespace MoDi.Desktop;

public static class TransportIdentity
{
    public const int AudioPort = 12345;
    public const int HandshakePort = 12347;
    public const string MdnsServiceType = "_modi._udp";
    public static readonly Guid BluetoothServiceUuid =
        Guid.Parse("6D6F4469-0001-4000-8000-000000000001");
}

File: android/app/src/main/java/com/modi/connect/core/TransportIdentity.kt
package com.modi.connect.core

import java.util.UUID

object TransportIdentity {
    const val AUDIO_PORT = 12345
    const val HANDSHAKE_PORT = 12347
    const val MDNS_SERVICE_TYPE = "_modi._udp"
    val BLUETOOTH_SERVICE_UUID: UUID = UUID.fromString("6D6F4469-0001-4000-8000-000000000001")
}
